### ISSUE TITLE
The creeper in on death: is incorrectly recognized as a firework

### ISSUE THREAD

**Issue by TIM31205** [SkriptLang/Skript#5812]
### Skript/Server Version

```markdown
[22:21:20 INFO]: [Skript] Skript's aliases can be found here: https://github.com/SkriptLang/skript-aliases
[22:21:20 INFO]: [Skript] Skript's documentation can be found here: https://skriptlang.github.io/Skript
[22:21:20 INFO]: [Skript] Skript's tutorials can be found here: https://docs.skriptlang.org/tutorials
[22:21:20 INFO]: [Skript] Server Version: git-Paper-69 (MC: 1.20.1)
[22:21:20 INFO]: [Skript] Skript Version: 2.6.4
[22:21:20 INFO]: [Skript] Installed Skript Addons:
[22:21:20 INFO]: [Skript]  - SkBee v2.14.1 (https://github.com/ShaneBeee/SkBee)
[22:21:20 INFO]: [Skript] Installed dependencies:
[22:21:20 INFO]: [Skript]  - Vault v1.7.3-b131
```

### Bug Description

The crepper is recognized as a firework.

### Expected Behavior

The crepper should be an entity.

### Steps to Reproduce

`on death:`
`    victim is creeper`
but
`on death of creeper:`
It's normal.

### Errors or Screenshots

[22:13:05 INFO]: Line 153: (server.sk)
[22:13:05 INFO]:     Can't compare an entity with a firework type
[22:13:05 INFO]:     Line: victim is creeper

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by APickledWalrus** [comment]
This is a known issue, and it's due to the fact that `creeper` is valid for entitytypes and firework types. Try using `victim is a creeper` :)

**Comment by TIM31205** [comment]
oh although I have searched for the same issus, I may not have found it because of the different keywords, thanks!

**Comment by TheLimeGlass** [comment]
This is fixed in 2.7.0-beta3
